OpenTabletDriver.UX/Windows/PluginManagerWindow.cs

```diff
@@ -144,7 +145,13 @@ protected async Task Install(string path)
 
         protected async Task Uninstall(DesktopPluginContext context)
         {
-            if (await App.Driver.Instance.UninstallPlugin(context.FriendlyName))
+            context.Directory.Refresh();
+            if (context.Directory.Exists && await App.Driver.Instance.UninstallPlugin(context.FriendlyName))
+            {
+                AppInfo.PluginManager.UnloadPlugin(context);
+                await Refresh();
+            }
+            else if (!context.Directory.Exists)
             {
                 AppInfo.PluginManager.UnloadPlugin(context);
                 await Refresh();
```

Comment:
Is there some reason we're doing it like this? Because if none then:
```suggestion
            if (context.Directory.Exists)
                await App.Driver.Instance.UninstallPlugin(context.FriendlyName)

            AppInfo.PluginManager.UnloadPlugin(context);
            await Refresh();
```

Reply:
The reason it is like that was if a manual delete was performed, however that would work with the downside of losing UI interactivity in the `else` block. Will clean this up to be more clear though.

Reply:
Ah, right. Can't see beyond what GitHub shows lol.